cernopendata/templates/cernopendata_theme/header.html

```diff
@@ -1,7 +1,7 @@
 {%- extends "invenio_theme/header.html" %}
 {%- block navbar %}
 <nav class="navbar navbar-expand-lg navbar-custom">
-  <a class="navbar-brand" href="{{url_for('cernopendata_pages.index')}}"><img src="{{url_for('static', filename='assets/img/logo-long.svg')}}" width="200" alt=""></a>
+  <a class="navbar-brand" href="{{url_for('cernopendata_pages.index')}}"><img src="{{url_for('static', filename='assets/img/logo_blue.png')}}" width="115" alt=""></a>
   <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
```

Comment:
Please remove the old logo, otherwise there will surely be a mix-up ;)

Also, it would be good to add presentable logo ("on-the-white-background") as part of the same PR.

Reply:
Added everything in an additional PR as this one was already merged.